src/aap_eda/services/activation/activation_manager.py

```diff
@@ -77,6 +77,11 @@ def __init__(
 
         self.container_logger_class = container_logger_class
 
+    def _update_started_time(self) -> None:
+        """Update latest instance's started_at to now."""
+        self.latest_instance.started_at = timezone.now()
+        self.latest_instance.save(update_fields=["started_at"])
+
```

Comment:
@bzwei do you also need to update the `updated_at` timestamp? which is also used in checking if is_unresponsive

Reply:
`updated_at` is only set when a heartbeat is received.

Reply:
Should we use a field ad-hoc for the heartbeat? updated_at is in the eda a field related to the DB record and its plausible to be updated on any other change not related to heartbeat. I feel we should track the heartbeat in a field explicitly dedicated for it.

Reply:
@Alex-Izquierdo sorry I missed your comment before the merge. updated_at is a particular field added for the heartbeat. The one for DB record and auto generated is normally called modified_at, which is not enable in this table.